core/kernel/console.c

```diff
@@ -27,6 +28,8 @@ void console_init(void)
 {
 	if (IS_ENABLED(CFG_SEMIHOSTING_CONSOLE))
 		semihosting_console_init(CFG_SEMIHOSTING_CONSOLE_FILE);
+	else if (IS_ENABLED(CFG_FFA_CONSOLE))
```

Comment:
It would be good to add `CFG_FFA_CONSOLE ?= n` to `mk/config.mk` with a comment explaining what it is and what is required to use it.

Reply:
Done